packages/editor/src/browser/navigation/navigation-location-service.ts

```diff
@@ -165,12 +169,13 @@ export class NavigationLocationService {
     }
 
     /**
-     * Clears the navigation history.
+     * Clears the editor history.
```

Comment:
`editor history` might be misleading:

```suggestion
     * Clears the total history.
```